### ISSUE TITLE
any reason not to print an error here?

### ISSUE THREAD

**Issue by meshula** [floooh/sokol#594]
I always locally patch this with a LOG of the error string. Is there a reason that's not the standard behavior?

```c
_SOKOL_PRIVATE bool _sg_validate_end(void) {
    if (_sg.validate_error != _SG_VALIDATE_SUCCESS) {
        #if !defined(SOKOL_VALIDATE_NON_FATAL)
            SOKOL_LOG("^^^^  SOKOL-GFX VALIDATION FAILED, TERMINATING ^^^^");
            SOKOL_LOG(_sg_validate_string(_sg.validate_error));   /// <<<< I always add this
            SOKOL_ASSERT(false);
        #endif
        return false;
    }
    else {
        return true;
    }
}
```

**Comment by meshula** [comment]
My bad, the messages do appear in the log, I was somehow not spotting them. sorry for the noise.

**Comment by floooh** [comment]
Yep I'm aware that the validation messages are not very readable at the moment. I didn't have a great idea so far how to improve it though.

**Comment by meshula** [comment]
To resolve the problem I was chasing, which was an "empty or non-continugous attrs" assertion, I ended up dumping all the attrs to the console upon failure.

I am definitely appreciative of the amount of checking in the library currently, regardless of how easy or hard it is to read the console output!

(The root cause of the problem I was chasing was that I have a hybrid C/Zig project, and I needed to declare the preproc for zig padding in my C code, so I was sending bad color attachments in my desc. I experimented to see if the padding is still necessary in 0.9.x, sadly it is still necessary.)
